reporter/src/main/kotlin/reporters/AsciidocTemplateReporter.kt

```diff
@@ -81,32 +84,44 @@ class AsciidocTemplateReporter : Reporter {
     override fun generateReport(input: ReporterInput, outputDir: File, options: Map<String, String>): List<File> {
         val asciidoctorAttributes = AttributesBuilder.attributes()
 
-        options[OPTION_PDF_THEME_PATH]?.let { themePath ->
-            File(themePath).also {
-                require(it.isFile) { "Could not find pdf-theme file at '${it.absolutePath}'." }
-            }
+        // Also see https://github.com/asciidoctor/asciidoctorj/issues/438 for supported backends.
+        val backend = options[OPTION_BACKEND] ?: BACKEND_PDF
+
+        if (backend.equals(BACKEND_PDF, ignoreCase = true)) {
```

Comment:
While currently acceptable, the decision logic related to the backend seems a bit clumsy. If more special cases are added, we could think about having a more generic concept, such as a strategy.

Reply:
I agree it's a bit hacky, also to work with strings instead of e.g. enums, but I also agree it's probably acceptable for now, and has the benefit that we do not need to maintain this code for newly added backends to Asciidoc.